### PR TITLE
Solves #179

### PR DESCRIPTION
**Pull request by prajapati-parth**

### CODE REVIEW

**Comment by AllenFang** [comment]
@prajapati-parth thanks your contribution, I'll check it later. In addition, could you please change the base branch to `develop` branch.  `master` branch is just for release, thank you !!!

File: packages/react-bootstrap-table2/src/row.js [hunk 1]
```diff
@@ -56,6 +57,15 @@ class Row extends Component {
     }
   }
 
+  handleSimpleRowClick(e) {
+    const {
+      rowIndex,
+      attrs
+    } = this.props;
+
+    attrs.onClick(e, rowIndex);
```
Comment (AllenFang) [inline packages/react-bootstrap-table2/src/row.js]: You can pass the `this.props.row`, in #179, we hope a row object be passed.

File: packages/react-bootstrap-table2/src/row.js [hunk 2]
```diff
@@ -89,6 +99,8 @@ class Row extends Component {
     const trAttrs = { ...attrs };
     if (clickToSelect) {
       trAttrs.onClick = this.handleRowClick;
+    } else {
+      trAttrs.onClick = this.handleSimpleRowClick;
```
Comment (AllenFang) [inline packages/react-bootstrap-table2/src/row.js]:
```js
if (clickToSelect) {
  trAttrs.onClick = this.handleRowClick;
} else if (attrs.onClick) {
  // Only attach onClick if user defined
  trAttrs.onClick = this.handleSimpleRowClick;
}
```

File: packages/react-bootstrap-table2/src/row.js [hunk 3]
```diff
@@ -35,7 +36,7 @@ class Row extends Component {
 
     const clickFn = () => {
       if (attrs.onClick) {
-        attrs.onClick(e);
+        attrs.onClick(e, row, rowIndex);
```
Comment (prajapati-parth) [inline packages/react-bootstrap-table2/src/row.js]: Passed the `row` object.

**Comment by Johnny-jober** [comment]
@AllenFang if this change is released, can I use row object at rowEvents event? (currently only 'e' is available)
sorry to rush you but, when is the rough release schedule?
Thanks a lot for this wonderful library.

**Comment by AllenFang** [comment]
@Johnny-jober sure, next week we will release!

**Comment by AllenFang** [comment]
New Release: https://react-bootstrap-table.github.io/react-bootstrap-table2/blog/2018/02/14/version-bump.html

Let me know if theres any problem, thanks

**Pull request merged**